Resources/Prototypes/SimpleStation14/Loadouts/traits.yml

```diff
@@ -24,3 +24,19 @@
     components:
       - Nearsighted
   item: ClothingEyesHudMedicalPrescription
+
+- type: loadout
+  id: LoadoutNearsightedEyesSecurityGlasses
+  name: Prescription Security Glasses
+  description: Now you can see AND be stylish!
+  category: Traits
+  cost: 4
+  jobWhitelist:
+    - HeadOfSecurity
+    - SecurityCadet
+    - SecurityOfficer
+    - Warden
+  whitelist:
+    components:
+      - Nearsighted
```

Comment:
I wonder if this works as intended (if it doesn't I'm going to explode, hold on)

Reply:
alright I am good coder man